src/components/currency/Home.js

```diff
@@ -0,0 +1,30 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import Exchange from './Exchange';
+import { filterdata } from '../../redux/exchangeSlice';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const { exchange } = useSelector((state) => state.exchange);
+  const limitValues = Object.values(exchange).slice(0, 200);
+  const [result, setResult] = useState('');
+
+  const search = (e) => {
+    const { value } = e.target;
+    const filterData = limitValues.filter((item) => item.currencyCode === value.toUpperCase());
```

Comment:
- [ ] [OPTIONAL] Instead of searching for countries' currency codes to be exactly the same as the entered string. You can use the `contains` method which finds all occurrences of the entered string.